lib/cli/src/frameworks/vue/Page.stories.js

```diff
@@ -1,25 +1,28 @@
+import { within, userEvent } from '@storybook/testing-library';
+
 import MyPage from './Page';
 import * as HeaderStories from './Header.stories';
 
 export default {
   title: 'Example/Page',
   component: MyPage,
+  parameters: {
+    // More on Story layout: https://storybook.js.org/docs/vue/configure/story-layout
+    layout: 'fullscreen',
+  },
 };
 
-const Template = (args, { argTypes }) => ({
-  props: Object.keys(argTypes),
+const Template = () => ({
   components: { MyPage },
-  template:
-    '<my-page :user="user" @onLogin="onLogin" @onLogout="onLogout" @onCreateAccount="onCreateAccount" />',
+  template: '<my-page />',
 });
 
-export const LoggedIn = Template.bind({});
-LoggedIn.args = {
-  // More on composing args: https://storybook.js.org/docs/vue/writing-stories/args#args-composition
-  ...HeaderStories.LoggedIn.args,
-};
-
 export const LoggedOut = Template.bind({});
-LoggedOut.args = {
-  ...HeaderStories.LoggedOut.args,
+
+// More on interaction testing: https://storybook.js.org/docs/vue/writing-tests/interaction-testing
+export const LoggedIn = Template.bind({});
+LoggedIn.play = async ({ canvasElement }) => {
+  const canvas = within(canvasElement);
+  const loginButton = await canvas.getByRole('button', { name: /Log in/i });
```

Comment:
Seems like using `findBy` instead fixes the issue, however it still fails when refreshing the browser. It doesn't fail when visiting the story.

```suggestion
  const loginButton = await canvas.findByRole('button', { name: /Log in/i });
```